lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -3795,10 +3795,7 @@ static void recordConformanceDependency(DeclContext *DC,
       Conformance->getDeclContext()->getParentModule())
     return;
 
-  // FIXME: 'deinit' is being used as a dummy identifier here. Really we
-  // don't care about /any/ of the type's members, only that it conforms to
-  // the protocol.
-  tracker->addUsedMember({Adoptee, DeclBaseName::createDestructor()},
+  tracker->addUsedMember({Adoptee, Identifier()},
```

Comment:
Unless you're planning on scrapping ReferenceNameTracker entirely, why not introduce an addUsedNominal() method or something, which for now just does this?

Reply:
We're scrapping it. This PR is just to get it to do something reasonable before I start emulating it.